Composer/plugins/mockRemotePublish/src/index.ts

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
If wondering if instead of removing this we can keep it around as a test fixture?

Reply:
Works for me but we will have to disable it so it doesn't appear in the dropdown.

Reply:
disable in package.json